vue/components/ui/molecules/select-checkboxes/select-checkboxes.vue

```diff
@@ -170,6 +169,21 @@ export const SelectCheckboxes = {
             },
             deep: true
         }
+    },
+    methods: {
+        scrollToInput(keyBuffer) {

```

Comment:
`scrollToCheckbox`?

Reply:
Yeah, I didn't like the name I had either, yours is more appropriate.